gnomad/resources/grch37/reference_data.py

```diff
@@ -96,6 +96,23 @@
     },
 )
 
+clinvar_pathogenic = VersionedTableResource(
+    default_version="20181028",
+    versions={
+        "20181028": TableResource(
+            path="gs://gnomad-public/resources/grch37/clinvar/clinvar_pathogenic_20181028.ht",
+            import_func=import_sites_vcf,
```

Comment:
It might be best to add the pathogenic filtering function here instead to directly filter from the HT. Maybe we can add the filtering function to resource utils?